api/v1/cluster_webhook.go

```diff
@@ -2338,6 +2339,51 @@ func (gcs *GoogleCredentials) validateGCSCredentials(path *field.Path) field.Err
 	return allErrors
 }
 
+func (r *Cluster) validateManagedServices() field.ErrorList {
```

Comment:
we could potentially forbid the service names that could be used by the operator for the default services, see my other comment for more details

Reply:
I agree. Much simpler to manage